CMakeLists.txt

```diff
@@ -62,6 +62,11 @@ endif()
 set(CMAKE_C_STANDARD_REQUIRED TRUE)
 add_definition(ZENOH_C_STANDARD=${CMAKE_C_STANDARD})
 
+# Useful for CXX platforms that don't provide std::atomic
+if(CXX_ATOMIC_FALLBACK)
```

Comment:
Could you validate this using `check_cxx_source_compiles`? In that way, the fallback would happen automatically without the need for the user to specify this flag.

Reply:
Updated to use `check_cxx_source_compiles` it works on my machine, but it seems that the CI doesn't expose it